middleware/setBananaI18n.js

```diff
@@ -0,0 +1,61 @@
+const url = require('url');
```

Comment:
this is currently pretty much a copy/paste from the FE i18n.js file in terms of locale detection + the map of locales / files, just exported as node middleware instead.